### PR TITLE
First version homepage with clearer text

### PR DESCRIPTION
**Pull request by elektrobohemian**
affects only my text :suspect: 

@stefanfk : do you like it :baby_chick:

### CODE REVIEW

**Comment by stefanfk** [comment]
lesen ja, schreiben nein?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
 # elektrobohemian.github.io
 
 browser-based update
+
+## the new website
+
+i want to work in isolation
+
+### why?
+
```
Comment (stefanfk) [inline README.md]: it is because of ...

File: README.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
 # elektrobohemian.github.io
 
 browser-based update
+
+## the new website
+
+i want to work in isolation
+
+### why?
```
Comment (stefanfk) [inline README.md]: sllalalslselel

File: README.md [hunk 3]
```diff
@@ -1,3 +1,13 @@
 # elektrobohemian.github.io
```
Comment (stefanfk) [inline README.md]: comment to line 1

File: README.md [hunk 4]
```diff
@@ -1,3 +1,13 @@
...
+
+## the new website
+
+i want to work in isolation
+
+### why?
+
+* they make us to make silly changes
+* I am falling asleep
+* why, why, why
```
Comment (elektrobohemian) [inline README.md]: @stefanfk Kannst Du das lesen?

**Pull request merged**